### ISSUE TITLE
Fix heading order

### ISSUE THREAD

**Issue by atsansone** [dart-lang/site-www#5299]
Review if all HTML headings on all pages appear in sequence. This violates WCAG [G141](https://www.w3.org/TR/WCAG20-TECHS/G141).

**Comment by atsansone** [comment]
Ran external tool to check Heading order. Results: https://www.onairseo.com/crawl-report/4713

No errors in ordering, but some SEO issues with H1 heading length.

Closing as complete for accessibility.
